src/ingameop.cpp

```diff
@@ -244,7 +242,7 @@ static bool _intAddInGameOptions()
 	addIGTextButton(INTINGAMEOP_OPTIONS, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_OP_W, _("Options"), OPALIGN);
 	row++;
 
-	if (!s)
+	if ((!bMultiPlayer || !NetPlay.bComms) && !bInTutorial)
 	{
 		if (!bMultiPlayer)
```

Comment:
You can swap the contents of the if and else branches and simplify the condition to bMultiPlayer

Reply:
done